src/less/select/ds6/select.less

```diff
@@ -48,3 +48,19 @@ span.select__icon {
         color: @ds6-color-g204-gray;
     }
 }
+
+.select--large {
+    font-size: 18px;
```

Comment:
This has to move inside the `> select { ... }` block or it won't have the right specificity, and get's overridden by another selector.